news/_posts/2019-09-10-the-review-process.md

```diff
@@ -0,0 +1,89 @@
+
+So as you can see, in a Pull Request we review a lot of things ! To do all of these checks, we do not only look at the code on github. We sometimes pull the branch on our own computer to test and navigate the code in order to make sure all of these requirements are met. We discuss with team members about parts we are not sure. We can even sometimes run benchmarks or audits for critical pieces of code.
+
+Some Pull Requests are very easy to review because they are related to a part of the code we know very well, other instead are very hard to review because they are related to a complex topic, a complex area of the code or have a huge global impact on the software that is very hard to estimate and assess. These last reviews can take hours or days as we try our very best to make sure every contribution merged in the project meets the level of quality we want for it.
+
+Most of the time, if an issue is found during the review, the Core Team member will provide feedback about the issue and requests the author to modify the parts of the Pull Request that cannot be accepted like it. After the author of the Pull Request have implemented the requested changes, then the Pull Request can be approved and move forward to the next step.
+
+### It is not only technical
+
+Also, for some Pull Requests, we might look for help and ask people of the PrestaShop Product Team:
+- for Pull Requests that have a huge impact of the UX, we ask the UX design team to review it
+- for Pull Requests that introduce a big change in the behavior (either an existing feature or a new feature), we ask the Product team to review it
+- for Pull Requests that introduce new wordings (labels, titles, error and information messages), we ask the Content team to review it
+
+When the Pull Request has been validated by all of these people, it finally is checked by the QA team. The QA team will check that the Pull Request behavior is correct and also that existing features and behaviors are not modified/broken by the Pull Request.
+
+When the Pull Request has finally passed the QA validation, it is finally merged in the project and the author becomes (if he was not before) an opensource contributor to this great project :) .
+
+![Merge button on github](/assets/images/2019/09/Merge-Button.png)
+
+## Conclusion
+
+As you can see, the process between the submission of Pull Request and its merge is a long journey. This journey is not meaningless: the goal is to make sure we build a consistent and reliable software in the end, and when you consider the size of the software, it actually does not look too much !
```

Comment:
```suggestion
As you can see, the process between the submission of Pull Request and its merge is a long journey. This journey is not meaningless: the goal is to make sure we build a consistent and reliable software in the end, and when you consider the size of this project, it actually does not look too much!
```

I didn't quite understand what you menant by: "it actually does not look too much!"